Readme.md

```diff
@@ -21,9 +21,16 @@ To build and run this example the following requirements are necessary:
 
 TX and RX pin definitions are target specific - the code and the physical board must both be using the same pins:
 
-1. To know which pins the code is looking for, see the ``serial_asynch.cpp`` file for this sample. The pins are defined as ``TEST_SERIAL_ONE_TX_PIN`` and ``TEST_SERIAL_TWO_RX_PIN``.
+1. To know which pins the code is looking for, see the ``serial_asynch.cpp`` file for this sample. The pins are defined as ``YOTTA_CFG_HARDWARE_TEST_PINS_SERIAL_TX`` and ``YOTTA_CFG_HARDWARE_TEST_PINS_SERIAL_RX``.
 
-2. To know where on your board those pins are, please look at its [pinout picture](http://developer.mbed.org/platforms/FRDM-K64F/#overview).
+2. These pin definitions are set in the yotta target's ``target.json`` file which can be found at ``yotta_targets/frdm-k64f-gcc/target.json`` in this example.  Each keyword in the define represents a level in the target descriptor structure.  Under config...hardware...test-pins...serial, the pin port numbers are defined.
+```
+"serial": {
+"tx": "PTC17",
+"rx": "PTD2"
+}
+```
+3. To know where on your board those pins are, please look at its [pinout picture](http://developer.mbed.org/platforms/FRDM-K64F/#overview).
```

Comment:
Can you fix the formatting of the code (indentation) and then 3rd is not displayed properly.